config/browse_everything_providers.yml

```diff
@@ -0,0 +1,26 @@
+#
+# To make browse-everything aware of a provider, uncomment the info for that provider and add your API key information.
+# The file_system provider can be a path to any directory on the server where your application is running.
+#
+---
+file_system:
```

Comment:
Can you add an `if` around file_system so it's only used for `development` until we come up with a solution to use it in kubernetes/AWS?

Reply:
done